### PR TITLE
fix:removed fix width for superadmin

### PR DESCRIPTION
**Pull request by Vayras**
With this pull request the entire ui for superadmin frontend will scale to the screen , previously was set to fixed width

### CODE REVIEW

File: frontend/app/src/pages/superadmin/index.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import AdminAccessDenied from './accessDenied';
 const Container = styled.body`
   height: 100vh; /* Set a fixed height for the container */
   overflow-y: auto; /* Enable vertical scrolling */
-  width: 1366px;
+  /* width: 1366px; */
```
Comment (kevkevinpal) [inline frontend/app/src/pages/superadmin/index.tsx:13]: just delete it instead of commenting it out

**Comment by ecurrencyhodler** [comment]
Some context for this pr. 

https://github.com/stakwork/sphinx-tribes/pull/1085#issuecomment-1866785287

**Comment by Vayras** [comment]
@ecurrencyhodler With this PR , as per the designer and as mentioned I have removed the fixed width for the bounties superadmin page

**Comment by kevkevinpal** [comment]
lgtm @ecurrencyhodler we should think about a mobile view for the admin page since it looks like this after a certain breakpoint 
Screenshot 2023-12-23 at 9 59 24 PM

And like this on mobile

Screenshot 2023-12-23 at 9 59 46 PM

**Pull request merged**